plugins/woocommerce/client/legacy/js/admin/woocommerce_admin.js

```diff
@@ -361,6 +361,26 @@
 			$( this ).trigger( 'focus' );
 		} );
 
+		const setPostThumbnail = $( '#set-post-thumbnail' );
+
+		if ( setPostThumbnail ) {
+			setPostThumbnail
+				.after(
+					`<span class="woocommerce-help-tip" tabindex="-1" for="content" aria-label="${ woocommerce_admin_meta_boxes.i18n_product_image_tip }"></span>`
```

Comment:
Is `for` a valid attribute for a `<span>`? 

I only see a reference to it being used on `<label>`.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attr-for

Actually, since the `aria-label` is for the `<span>` itself, I don't think we need a `for` in any case... 

The example of `aria-label` on MDN shows it being used on a `<button>` that does not have text content. That is the same case we have here.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label